cmd/containerd-nydus-grpc/pkg/command/flags.go

```diff
@@ -127,31 +93,13 @@ func buildFlags(args *Args) []cli.Flag {
 			Usage:       "whether to enable nydus-overlayfs",
 			Destination: &args.EnableNydusOverlayFS,
 		},
-		&cli.BoolFlag{
-			Name:        "enable-stargz",
```

Comment:
Does this impact compatibility?

Reply:
I suppose it doesn't. Stargz compatibility does not work for a long time.

Reply:
Should other flags still be reserved to ensure that snapshotter can be started if users use old flags?

Reply:
I am inclined not to reserve other parameters. We have to migrate to the configuration file pattern in the end.
This PR only removes the parameters that are not likely to be used.

Reply:
Ok, we need notice users that this is a breaking change in the release note.